src/components/introduction.stories.mdx

```diff
@@ -52,7 +52,12 @@ const Root = () => (
 
 3. Join a meeting
 
-To join a meeting, you will need to pass the meeting and attendee data from Chime's [CreateAttendee](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html) and [CreateMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html) APIs. [Read this for more details](https://aws.github.io/amazon-chime-sdk-js/modules/gettingstarted.html#meeting-service).
+To join a meeting, you will need to:
+
+  * Fetch the meeting and attendee data from Chime's [`CreateAttendee`](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html) and [`CreateMeeting`](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html) APIs
+      * See the [meeting join example](https://github.com/aws-samples/amazon-chime-sdk/blob/daefd28611a577877146a46aa189958c5bd0234b/apps/meeting/src/containers/MeetingForm/index.tsx#L70-L85), the [API calls example](https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/meeting/server.js), and [server application example](https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/meeting/server.js) for more details
+  * Create a `MeetingSession` using the above data
```

Comment:
If a code sample is provided, you can remove these radios.

Reply:
Add link to this example: https://github.com/aws/amazon-chime-sdk-js#getting-responses-from-your-server-application